exec/disk_fill.go

```diff
@@ -49,6 +49,10 @@ func NewFillActionSpec() spec.ExpActionCommandSpec {
 					Name: "percent",
 					Desc: "Total percentage of disk occupied by the specified path. If size and the flag exist, use this flag first. The value must be positive integer without %",
 				},
+				&spec.ExpFlag{
```

Comment:
I looked at this flag not being used in the disk fill experiment.

Reply:
yes，I forget push it.  i will push it again